tests/rpc/spec/drawing_spec.js

```diff
@@ -0,0 +1,93 @@
+describe('Drawing', function(){
+
+    function handleEvent(name, handler) {

```

Comment:
This function should be in some common helpers file.

Reply:
Will move.